### PR TITLE
Netsuite: support fields with CDATA type. The field names are taken from arbitrary places in NS documentation and not automatically

### PR DESCRIPTION
**Pull request by omrilit**

### CODE REVIEW

File: packages/netsuite-adapter/test/transformer.test.ts [hunk 1]
```diff
@@ -15,10 +15,12 @@
 */
 import { InstanceElement } from '@salto-io/adapter-api'
 import { createInstanceElement, toCustomizationInfo } from '../src/transformer'
-import { ENTITY_CUSTOM_FIELD, SCRIPT_ID } from '../src/constants'
+import { ADDRESS_FORM, ENTITY_CUSTOM_FIELD, SCRIPT_ID } from '../src/constants'
 import { customTypes } from '../src/types'
 import { convertToCustomizationInfo, convertToXmlContent } from '../src/client/client'
 
+const uglify = (xmlContent: string): string => xmlContent.replace(/\n\s*/g, '')
```
Comment (ori-moisis) [inline packages/netsuite-adapter/test/transformer.test.ts:22]: `removeWithspace`?

File: packages/netsuite-adapter/src/constants.ts [hunk 2]
```diff
@@ -17,16 +17,21 @@ export const NETSUITE = 'netsuite'
 export const RECORDS_PATH = 'Records'
 export const TYPES_PATH = 'Types'
 export const SUBTYPES_PATH = 'Subtypes'
+export const FIELD_TYPES_FILENAME = 'fieldTypes'
```
Comment (ori-moisis) [inline packages/netsuite-adapter/src/constants.ts:20]: other constants are `PATH`, why is this `FILENAME`?

File: packages/netsuite-adapter/src/constants.ts [hunk 3]
```diff
@@ -17,16 +17,21 @@ export const NETSUITE = 'netsuite'
...
+export const ADDRESS_FORM = 'addressForm'
 
 // Type Annotations
 export const SCRIPT_ID_PREFIX = 'scriptIdPrefix'
 
 // Fields
 export const SCRIPT_ID = 'scriptid'
 
+// Field Types
+export const CDATA = 'cdata'
```
Comment (ori-moisis) [inline packages/netsuite-adapter/src/constants.ts:33]: just noticed I forgot to comment about this, I don't understand why we need this constant

File: packages/netsuite-adapter/src/constants.ts (hunk 2 above)
Comment (omrilit) [reply packages/netsuite-adapter/src/constants.ts:20]: fieldTypes is the filename. its path is:
`Records/Types/Subtypes/fieldTypes.nacl`

Do you still think it should be renamed?

File: packages/netsuite-adapter/test/transformer.test.ts (hunk 1 above)
Comment (omrilit) [reply packages/netsuite-adapter/test/transformer.test.ts:22]: Just wanted to test whether you really read the code :)
It's not exactly remove whitespaces as it removes only whitespaces in the start of new lines.
maybe `minify` / `unformat` / `toOneLineXml` ?
WDYT?

Comment (ori-moisis) [reply packages/netsuite-adapter/test/transformer.test.ts:22]: `removeLineBreaks`?
it doesn't just remove the whitespace, it removes all `\n` as well

File: packages/netsuite-adapter/src/constants.ts (hunk 2 above)
Comment (ori-moisis) [reply packages/netsuite-adapter/src/constants.ts:20]: I wouldn't have created a new suffix for it (would have kept on with `PATH`) but up to you, it is a very small nitpick

**Pull request merged**